packages/api/db/migration/20230124_add_wage_do_not_ask_again_column_to_userFarm.js

```diff
@@ -0,0 +1,30 @@
+/*
+ *  Copyright 2019, 2020, 2021, 2022 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+export const up = function (knex) {
+  return Promise.all([
+    knex.schema.table('userFarm', (table) => {
```

Comment:
In both up and down functions consider using the alterTable() knex function instead of table() for readability.